automation/system_test/prepare.py

```diff
@@ -326,49 +369,77 @@ def main():
 @main.command(context_settings=dict(ignore_unknown_options=True))
 @click.argument("mlrun-version", type=str, required=True)
 @click.option(
-    "--mlrun-repo", "-mr", default=None, help="Override default mlrun image repository."
+    "--override-image-registry",
+    "-mreg",
```

Comment:
I guess it should be this way no ?
```suggestion
    "-oir",
```

Reply:
"-oireg"

Reply:
-oireg